rust/runtime/src/errors.rs

```diff
@@ -4,16 +4,12 @@ use alloc::alloc;
 use std::alloc;
 use std::num;
 
+use common::errors as coerrors;
```

Comment:
`common_errors`. `coerrors` has a totally different meaning

Reply:
Hmm! interesting. Do we have *co*error in any sense? just curious.

Reply:
> coerror in any type theoretic sense

Not sure! _co_ is associated with _covariance_ or _cooperative_ in many other contexts, which made it potentially confusing here. Fortunately characters are cheap when rustfmt gives you a roomy 100 per line :)